baitap-22-10-2020/main.js

```diff
@@ -0,0 +1,101 @@
+// Bài 1
+const printMultiplication = number => {
+    if (number >= 1 && number <= 10) {
+        for(let i = 1; i <= 10; i++) {
+        console.log(number + ' * ' + i + ' = ' + (number * i))
+        } 
+    } else console.log('Chỉ nhận giá trị từ 1 đến 10')
+}
+
+// Bài 2
+const printEvenNumbers = number => {
+    if (number >= 1 && number <= 30) {
+        for(let i = 1; i <= number; i++) {
+            if (i % 2 === 0) console.log(i)
+        } 
+    } else console.log('n chỉ nhận giá trị từ 1 đến 30')
+}
+
+// Bài 3
+const getSum = number => {
+    let sum = 0
+    if (number >= 1 && number <= 30) {
+        for(let i = 1; i <= number; i++) {
+             sum += i
+        } 
+        return sum
+    } else console.log('n chỉ nhận giá trị từ 1 đến 30')
+}
+// Bài 4
+const getFactorial = number => {
+    let factorial = 1
+    if (number >= 1 && number <= 30) {
+        for(let i = 1; i <= number; i++) {
+            factorial *= i
+        } 
+        return factorial
+    } else console.log('n chỉ nhận giá trị từ 1 đến 30')
+}
+
+// Bài 5
+const countEvenNumbers = array => {
+    let count = 0
+    for (let i = 0; i < array.length ; i++) {
+        if (array[i] % 2 === 0) {
+            count += 1
+        }
+    } 
+    return count
+}
+
+// Bài 6
+array = ['A', 'C', 'A', 'A', 'B', 'D', 'B']

```

Comment:
Khi khai báo bất cứ thứ gì, đều phải thêm từ khóa khai báo vào
Trong trường hợp này hãy đặt là `const`
```suggestion
const array = ['A', 'C', 'A', 'A', 'B', 'D', 'B']

```